### DISCUSSION TITLE
Klaviyo integration to Elementor Forms

### DISCUSSION THREAD

**Discussion by mikkelaveo** [elementor/elementor#20058] [Feature Request]
Hi Elementor,

Can you implement an integration to Klaviyo (https://www.klaviyo.com/) to Elementor Forms like to once you have made for MailChimp and Active Campaign?

Currently the only way to use Klaviyo in Elementor is with the Embed Forms from Klaviyo or this plugin: https://wordpress.org/plugins/include-klaviyo-for-elementor-pro/. 

But the third part plugin only supports the "email" field.

Best regards,
Mikkel

**Comment by nolderoos** [comment]
+1

**Comment by costrut** [comment]
+1 - Could the Elementor team please get the ball rolling on this? Thank you so much!

**Comment by prizewhisky** [comment]
I won't hold my breath as this was first requested in 2022... But plus one on this as your mailchimp integration was very useful but moved over platforms.
